compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx

```diff
@@ -7,15 +7,21 @@ content:
   paragraph: This guide provides step-by-step instructions to help you migrate your existing Block Storage volumes and snapshots to Scaleway's new Block Storage management system.
 tags: instance snapshot volume b_ssd sbs
 dates:
-  validation: 2024-10-10
+  validation: 2024-11-15
   posted: 2024-08-07
 categories:
   - compute
 ---
 
-Scaleway is transitioning the management of Block Storage volumes and snapshots from compute teams to storage teams to enhance performance and reliability.
-While the characteristics of your migrated volumes and snapshots will remain unchanged, Scaleway's new storage management system will now manage them.
-This guide provides the steps needed to migrate your volumes and snapshots using either the [Scaleway Command Line Interface (CLI)](/developer-tools/scaleway-cli/quickstart/) tool (recommended) or via the [Instances API](https://www.scaleway.com/en/developers/api/instance/#path-volumes-migrate-a-volume-andor-snapshots-to-sbs-scaleway-block-storage).
+To enhance performance and reliability, Scaleway is transitioning the management of Block Storage volumes and snapshots from Compute to Storage.
+
+The migration process ensures that the characteristics of your volumes and snapshots remain unchanged, even for volumes attached to running Instances.
+
+After migration, these resources will be managed under [Storage](/storage/block/quickstart/) instead of Compute.
+
+During migration, unified snapshots will be converted into Block Storage snapshots and removed from Compute. If a unified snapshot is part of an image, it will be replaced by the corresponding Block Storage snapshot.
```

Comment:
Likely we should state that unified snapshot will be converted to SBS block storage snapshot type (not sure how it should be called here).

Reply:
I've updated the wording and added a link to the new Block Storage